Apps/APN/ios/Podfile

```
@@ -0,0 +1,43 @@
+require_relative '../node_modules/react-native/scripts/react_native_pods'
+require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+
+platform :ios, '13.0'
+install! 'cocoapods', :deterministic_uuids => false
+
+target 'SampleApp' do
+  config = use_native_modules!
+
+  # Flags change depending on the env values.
+  flags = get_default_flags()
+
+  pod 'CustomerIO/MessagingPushAPN', '2.1.2'
```

```suggestion
  pod 'CustomerIO/MessagingPushAPN', '~> 2'
```

Our docs suggest this syntax.

---

I suggest that we add [these functions](https://github.com/customerio/amiapp-reactnative/blob/d1f74c6478fc1b9cf4eb49b2b500cf997a5f0ef8/ios/Podfile#L20-L23) to this file to make development easier for future mobile engineers who want to install pre-production versions of the native iOS SDK into the sample app.

---

Great suggestion. I would like to create another PR for this change to avoid conflicts in other PRs as this branch is a base to others.

---

> Our docs suggest this syntax.

Updated this another commit as syntax update was also required for NSE.